tests/epics/signal/test_signals.py

```diff
@@ -897,5 +897,5 @@ async def test_signals_created_for_prec_0_float_can_use_int(ioc: IOC):
 async def test_signals_created_for_not_prec_0_float_cannot_use_int(ioc: IOC):
     pv_name = f"{ioc.protocol}://{PV_PREFIX}:{ioc.protocol}:float_prec_1"
     sig = epics_signal_rw(int, pv_name)
-    with pytest.raises(TypeError):
+    with pytest.raises(TypeError, match="[ca:float_prec_1 has type float not int]"):
```

Comment:
``` suggestion
    with pytest.raises(TypeError, match="ca:float_prec_1 has type float not int"):
```

Ah sorry, I didn't mean literally include `[]`. Pretty sure this regex will allow any error message containing one of the characters inside:

``` python
>>> with pytest.raises(ValueError, match="[hi there]"):
...     raise ValueError("a")
...
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError: a

...
AssertionError: Regex pattern did not match.
 Regex: '[hi there]'
 Input: 'a'
>>> with pytest.raises(ValueError, match="[hi there]"):
...     raise ValueError("e") # passes
... 
```

Reply:
Ah yeah, whoopsy